txnkv/transaction/2pc.go

```diff
@@ -786,8 +786,8 @@ func txnLockTTL(startTime time.Time, txnSize int) uint64 {
 	return lockTTL + uint64(elapsed)
 }
 
-var preSplitDetectThreshold uint32 = 100000
-var preSplitSizeThreshold uint32 = 32 << 20
+var preSplitDetectThreshold uint32 = 1000000
```

Comment:
How about making these optional and changing the default value in the tidb-cse?

Reply:
That would be much more work to do.